### PR TITLE
Version 4.6.8 changelog

### PR DESCRIPTION
**Pull request by snakefoot**
Example: https://github.com/snakefoot/NLog/blob/NLogVer468/CHANGELOG.md

todo:

- [x] https://github.com/NLog/NLog/pull/3662
- [x] https://github.com/NLog/NLog/pull/3660

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,40 @@ Date format: (year/month/day)
 
 ## Change Log
 
+### V4.6.8 (2019/11/03)
```
Comment (304NotModified) [inline CHANGELOG.md]: todo update date :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,40 @@ Date format: (year/month/day)
 
 ## Change Log
 
+### V4.6.8 (2019/11/04)
```
Comment (304NotModified) [inline CHANGELOG.md]: todo add:

https://github.com/NLog/NLog/pull/3662 and https://github.com/NLog/NLog/pull/3660

**Comment by snakefoot** [comment]
Added last minute changes

**Comment by 304NotModified** [comment]
Thanks!

**Pull request merged**